index.bs

```diff
@@ -703,6 +708,11 @@ A {{WebTransport}} object has the following internal slots.
    <td class="non-normative">The number of concurrently open
    [=bidirectional=] streams the application anticipates the server creating, or null.
   </tr>
+  <tr>
+   <td><dfn>`[[Protocol]]`</dfn>
+   <td class="non-normative">A string indicating the subprotocol selected by the server,
```

Comment:
Do we have a reference for "subprotocol"?  I know that it's a websocket concept, so maybe that can be used.

Reply:
In WebSocket, the options-member is [protocols](https://websockets.spec.whatwg.org/#dom-websocket-websocket), but refers to each string as a _"subprotocol name"_ that has to _"match the requirements for elements that comprise the value of [`Sec-WebSocket-Protocol`](https://datatracker.ietf.org/doc/html/rfc6455#section-11.3.4) fields as defined by The WebSocket protocol. [[WSP]](https://websockets.spec.whatwg.org/#biblio-wsp)"_

<img width="1261" alt="image" src="https://github.com/w3c/webtransport/assets/3136226/52de7647-5780-4b6c-b59a-289de04c2c31">

WebSockets is mostly what I modeled this PR on. We could s/subprotocol/subprotocol name/ maybe if that helps?

I could also break out "subprotocol name" as a definition if that helps?